test/e2e/components/vl-accordion.js

```diff
@@ -47,6 +67,11 @@ class VlAccordion extends VlElement {
   async _accordionDiv() {
     return this.shadowRoot.findElement(By.css('div[data-vl-accordion]'));
   }
+
+  async isContentShown() {
```

Comment:
```suggestion
  async isContentDisplayed() {
```

Meer analoog aan naamgeving Selenium `isDisplayed`.